app/assets/javascripts/dialog.js

```diff
@@ -0,0 +1,8 @@
+function getDialog() {
```

Comment:
actually, this method is about styling you dialog, so no need to return dialog, also bad naming.